src/dmd/errors.d

```diff
@@ -42,7 +42,17 @@ enum Classification
  *      format = printf-style format specification
  *      ...    = printf-style variadic arguments
  */
-extern (C++) void error(const ref Loc loc, const(char)* format, ...)
+static if (__VERSION__ < 2092)
+    extern (C++) void error(const ref Loc loc, const(char)* format, ...)
+    {
+        _error(loc, format);
```

Comment:
Using `va_list` here makes the test pass again.
```suggestion
        va_list ap;
        va_start(ap, format);
        verror(loc, format, ap);
        va_end(ap);
```

Reply:
Thanks a lot!